### DISCUSSION TITLE
How to apply the style changes to the ID instead of the class

### DISCUSSION THREAD

**Discussion by bablilayoub** [GrapesJS/grapesjs#5225] [Q&A] [answered]
Hi,
I have two elements with the same classes. I want to apply style changes with style manager to one of these two elements. I add ID but the style applies to the class not to the ID.

What can I do?

**Comment by artf** [comment, marked as answer]
You can use [component-first selector option](https://grapesjs.com/docs/modules/Selectors.html#component-first-selectors)

**Reply by bablilayoub** [reply]
Thank you, its working
